game_map.py

```diff
@@ -28,6 +28,9 @@ def __init__(
         self.explored = np.full(
             (width, height), fill_value=False, order="F"
         )  # Tiles the player has seen before
+        self.aoe = np.full(
```

Comment:
If this is going to be added as a permanent buffer then a more generic name should be used.
```suggestion
        self.highlight = np.full(
```